charts/airflow/templates/worker/worker-statefulset.yaml

```diff
@@ -162,7 +162,39 @@ spec:
             {{- else }}
             - "exec airflow celery worker"
             {{- end }}
-          {{- if $volumeMounts }}
+          {{- if .Values.worker.livenessProbe.enabled }}
```

Comment:
You have made a typo in the value, it should be `workers.{...}` not `worker.{...}` (with an "s").

(This is what is causing the CI to fail)